doc/classes/Camera2D.xml

```diff
@@ -60,12 +54,6 @@
 				Returns the specified camera limit. See also [member limit_bottom], [member limit_top], [member limit_left], and [member limit_right].
 			</description>
 		</method>
-		<method name="make_current">
-			<return type="void" />
-			<description>
-				Make this the current 2D camera for the scene (viewport and layer), in case there are many cameras in the scene.
-			</description>
-		</method>
```

Comment:
Removing these methods breaks compatibility, so it can't be done in the stable 3.x branch.

Reply:
I added them back in.